crates/store/re_types/definitions/rerun/datatypes/video_timestamp.fbs

```diff
@@ -0,0 +1,24 @@
+namespace rerun.datatypes;
+
+/// Specifies how to interpret the `video_time` field of a [datatypes.VideoTimestamp].
+enum VideoTimeMode: ubyte{
+    /// Invalid value. Won't show up in generated types.
+    Invalid = 0,
+
+    /// Presentation timestamp in nanoseconds since the beginning of the video.
+    Nanoseconds = 1 (default),
+
+    // Future values: FrameNr
+}
+
+/// Timestamp inside a [archetypes.AssetVideo].
+struct VideoTimestamp (
```

Comment:
I would still love to eventually have `time_mode = FrameNumber`.

So calling this a `Timestamp` will create a bit of a strange naming pattern.

Reply:
coincidentally I brought this very thing up on Slack expressing a similar (but not very deep) concern. @emilk pointed out that frame numbers can be seen as a form timestamp unit in this context.
very open to other suggestions not sure on this one 🤷  


https://rerunio.slack.com/archives/C041NHU952S/p1725981749912279?thread_ts=1725981199.569259&cid=C041NHU952S

Reply:
Some ideas from ChatGPT

```
VideoPosition: Emphasizes that the struct represents a position in the video, which could be either in frames or seconds.
VideoMarker: Suggests a point in the video, adaptable to frames or time.
TimeOrFrame: Makes it clear that the struct can hold either a time value or a frame number.
FrameOrTimeStamp: Explicitly names the two modes.
VideoCoordinate: Represents a general position in time (seconds or frames) as coordinates often imply different dimensions.
MediaTimeUnit: Indicates the struct holds a unit of time in different formats, either frames or seconds.
PlaybackPosition: Focuses on the position in playback, where both time and frame counts are relevant.
TimelineIndex: Suggests an index along a timeline that can be expressed in either frames or time.
```